docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -356,6 +356,17 @@ Important parameters to configure regarding the captive portal are the following
 
 For some browsers, is it preferable to redirect the user to a specific URL instead of the URL the user originally intended to visit. For these browsers, the URL defined in `redirecturl` will be the one where the user will be redirected. Affected browsers are Firefox 3 and later.
 
+* Filters under *Configuration -> Portal Profile -> Portal Name -> Fitlers*
+
+It's possible to get a specific portal profile based on filters. Actually in Packetfence you have the following filters: Connection Type, Network, Node Role, Port, realm, SSID, Switch, Switch Port, URI and VLAN.
+
+Example with the most common one:
+
+* SSID: Guest-SSID
+* VLAN: 100
+
+CAUTION: Node role will take effect only with a 802.1x connection and if you use vlan_filters.
+
```

Comment:
I suggest to move this information under **Optional components -> Portal Profiles**, where we already talk about filters. Please update that section at the same time.